test/format/verify.c

```diff
@@ -137,12 +137,54 @@ table_mirror_fail_msg_flcs(WT_SESSION *session, const char *checkpoint, TABLE *b
       checkpoint ? " (checkpoint " : "", checkpoint ? checkpoint : "", checkpoint ? ")" : "");
 }
 
+/*
+ * position_cursor_before --
+ *     Place a cursor on the key directly preceding the target key. It's ok to return WT_NOTFOUND
+ *     from this function as any subsequent cursor next calls will begin at the start of the table.
```

Comment:
Good catch! That's an artifact from when I was handling the `WT_NOTFOUND` retcode in the calling functions instead of inside `position_cursor_before`. On review there's no need to return a retcode anymore so I've updated `position_cursor_before` to be a void function.